src/Tooling/Bit.Tooling.Templates/TodoTemplate/Api/Controllers/AuthController.cs

```diff
@@ -83,11 +88,49 @@ public async Task SendEmailConfirmLink(SendEmailConfirmLinkRequestDto sendConfir
         var result = await _fluentEmail
             .To(user.Email, user.DisplayName)
             .Subject(EmailStrings.ConfirmationLinkEmailSubject)
-            .UsingTemplateFromEmbedded("TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
-                                    new EmailConfirmationModel
+            .UsingTemplateFromEmbedded($"TodoTemplate.Api.Resources.EmailConfirmation.cshtml",
+                                    new EmailLinksModel
                                     {
                                         DisplayName = user.DisplayName,
-                                        ConfirmationLink = confirmationLink
+                                        Link = confirmationLink
+                                    },
+                                    assembly)
+            .SendAsync(cancellationToken);
+
+        if (!result.Successful)
+            throw new ResourceValidationException(result.ErrorMessages.ToArray());
+    }
+
+    [HttpPost("[action]")]
+    public async Task SendEmailForgotPasswordLink(SendEmailForgotPasswordLinkRequestDto sendConfirmLinkEmailRequest, CancellationToken cancellationToken)
```

Comment:
SendPasswordResetTokenEmail instead

Reply:
it's even better to rename it to `SendResetPasswordEmail`.
it's not just the token or a link, it's a complete Email containing a link with a token :)

and also it's a lot better to rename the other method to `SendConfirmationEmail`.